output/schema/validation-errors.json

```diff
@@ -1460,6 +1460,12 @@
       ],
       "response": []
     },
+    "text_structure.test_grok_pattern": {
+      "request": [
+        "Request: missing json spec query parameter 'ecs_compatibility'"
```

Comment:
Please can you add this to `specification/text_structure/test_grok_pattern/TestGrokPatternRequest.ts`. Then this new section should disappear from the validation errors.